mods/voidmulti.sc2mod/base.sc2data/gamedata/actordata.xml

```diff
@@ -48,6 +65,7 @@
         <On Terms="SupporterDestruction" Send="AnimBracketStop BSD"/>
         <HostForProps Subject="ShieldBatteryRechargeEx5@Beam" Scope="Caster" Actor="Find"/>
         <ModelFlags index="AllowHitTest" value="0"/>
+        <FogVisibility value="Hidden"/>
```

Comment:
**undocumented**: Hides the Shield Battery's recharge's impact model in the fog of war

I assume that it created a snapshot image in the fog when you loose vision of the recharged structure. Now this is gone, so only the structure's snapshot image is displayed in the fog of war